cost/superseded_instance/superseded_instance.pt

```diff
@@ -410,20 +416,38 @@ end
 policy "pol_aws_superseded_instance" do
   validate_each $ds_combined_instance_data do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} possible superseded instances"
-    detail_template <<-EOS
-# Superseded Instances
-
-| Resource UID | Name | Current Instance Type | Region | New Instance Type | VPC Required | ENA Required | Monthly Estimated Cost |
-| ------------ | ---- | --------------------- | ------ | ----------------- | ------------ | ------------ | ---------------------- |
-{{ range data -}}
-| {{ if .warn }}**_{{ end -}}{{ .resource_id }}{{ if .warn }}_**{{ end -}} | {{ .name }} | {{.instance_type}} | {{.region}} | {{.new_instance_type}} | {{ if .vpc_incompatible_move }} **_TRUE_** {{else}} false {{end}} |  {{ if .ena_incompatible }} **_TRUE_** {{else}} false {{end}} | {{.run_rate}} |
-{{ end -}}
-
-___
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
-EOS
     check logic_or(lt(to_n(val(item,"run_rate")),$param_threshold), lt(to_n(val(item,"run_rate")),$param_minimum_savings_threshold))
     escalate $report_instances
+    export do
+      resource_level true
+      field "id" do
+        label "Instance ID"
+      end
+      field "resource_id" do
+        label "Resource UID"
+      end
+      field "name" do
+        label "Name"
+      end
+      field "instance_type" do
+        label "Current Instance Type"
+      end
+      field "region" do
```

Comment:
region should be the first column and sorted.

Reply:
Moved the region to the top field. I not not sure how to sort the export. Can you point to some example policy which has done this on a the export?